### PR TITLE
fix(storybook): dotcom shell storybook react fix

### PR DESCRIPTION
**Pull request by IgnacioBecerra**
### Related Ticket(s)
#9118

### Description
This fixes the React Storybook for Dotcom Shell and its variation stories.

### Changelog

**Changed**

- the way knobs are being created in Dotcom Shell for React storybook

### CODE REVIEW

**Comment by IgnacioBecerra** [comment]
Just pulled the changes in the Masthead story -- I'll have to adjust the Story to make sure the changes work again.

File: packages/react/src/components/DotcomShell/__stories__/DotcomShell.stories.js [hunk 1]
```diff
@@ -22,280 +20,426 @@ const footerTypeOptions = {
...
+          'hide the locale button (disableLocaleButton)',
+          false,
+          'Footer'
+        ),
+      },
+    };
+  },
+
+  shortFooter: () => {
+    const { Masthead: mastheadKnobs } = mastheadStory.story.parameters.knobs;
```
Comment (ariellalgilmore) [inline packages/react/src/components/DotcomShell/__stories__/DotcomShell.stories.js:141]: Looks great! Just need to update these stories as well

```suggestion
```

File: packages/react/src/components/DotcomShell/__stories__/DotcomShell.stories.js [hunk 2]
```diff
@@ -22,280 +20,426 @@ const footerTypeOptions = {
...
+          'Footer'
+        ),
+        type: 'short',
+      },
+    };
+  },
+
+  shortFooterLanguageOnly: () => {
+    const languageOnly = true;
+    const { Masthead: mastheadKnobs } = mastheadStory.story.parameters.knobs;
```
Comment (ariellalgilmore) [inline packages/react/src/components/DotcomShell/__stories__/DotcomShell.stories.js:157]:
```suggestion
```

**Review by ariellalgilmore — COMMENTED** [review]
Looks great! just minor updates

File: packages/react/src/components/DotcomShell/__stories__/DotcomShell.stories.js [hunk 3]
```diff
@@ -5,13 +5,11 @@
...
-import { select, object } from '@storybook/addon-knobs';
+import { boolean, select, text, object } from '@storybook/addon-knobs';
 import Content from './data/content';
 import DotcomShell from '../DotcomShell';
-import { Micro as footerMicroStory } from '../../Footer/__stories__/Footer.stories.js';
-import { Default as footerStory } from '../../Footer/__stories__/Footer.stories.js';
-import { WithL1 as l1Story } from '../../Masthead/__stories__/Masthead.stories.js';
 import languageItems from '../../Footer/__data__/language-items.json';
+import mastheadLinks from '../../Masthead/__stories__/data/MastheadLinks';
 import { Default as mastheadStory } from '../../Masthead/__stories__/Masthead.stories.js';
```
Comment (ariellalgilmore) [inline packages/react/src/components/DotcomShell/__stories__/DotcomShell.stories.js:13]: sorry should have included this too, but we can remove this as well
```suggestion
```

**Review by ariellalgilmore — APPROVED** [review]
LGTM!

**Pull request merged**